js/script.js

```diff
@@ -787,10 +787,27 @@ function Add_EventHandlers_To_Document()
     })
 }
 
+function selectPalette() {
+    var nes = document.getElementById("palette-div");
+    var gameboy = document.getElementById("gameboy-palette-div");
+    var radioNES = document.getElementById("radioNES");
+    var radioGameboy =document.getElementById("radioGB");
+
+    if (radioNES.checked) {
+        nes.style.display = "block";
+        gameboy.style.display = "none";
+    }
+    else if (radioGameboy.checked) {
```

Comment:
Can you drop `if` to the next line so the `if`s are on the same char column?

Reply:
For sure. Looking back through the script, this seems more in line with what you already have. Does that seem better?